### ISSUE TITLE
Remove redundant second search input field after searching

### ISSUE THREAD

**Issue by holybiber** [4training/mediawiki-skins-ForTrainingNG#1]
After using the search function, the search display page shows two search bars:
https://designbeta.4training.net/?search=Vergebung&title=Special%3ASearch&fulltext=Search

The best solution would probably be to remove the search bar from the Special:Search page (patching the PHP of the code that generates the page?)

**Comment by Krutsch** [comment]
I agree, removing it from the template and populating the search above with the seachQuery, if it exists
